### PR TITLE
[styles] added tourism-information-visitor_centre and changed icon for tourism-information 

### PR DESCRIPTION
**Pull request by map-per**
Closes #5049 #4993

- Added `tourism=information` `information=visitor_centre` [[Wiki](https://wiki.openstreetmap.org/wiki/Tag:information%3Dvisitor_centre)] with the same icon and priority as `information=office`
- Changed tourism-information icon to an 'i' without background casing
- tourisn-information-office icon is unchanged

#### Screenshots

Bildschirmfoto vom 2023-09-03 08-11-07
https://www.openstreetmap.org/way/1171072785

| Light |  Bildschirmfoto vom 2023-09-07 09-12-50  |
|--------|--------|
| **Dark** |Bildschirmfoto vom 2023-09-07 09-13-10 | 

https://www.openstreetmap.org/node/4592840092

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -11567,6 +11567,11 @@ sk:3Turistické informačné centrum
 fa:دفتر گردشگری
 mr:पर्यटन कार्यालय
 
+tourism-information-visitor_centre|@tourism
+en:3Visitor Centre
+de:3Besucherzentrum
+es:Centro de visitantes
+
```
Comment (matheusgomesms) [inline data/categories.txt:11574]:
```suggestion
pt:Centro de visitantes
pt-BR:Centro de visitantes
```

**Review by pastk — COMMENTED** [review]
Looks good!

I think there is nothing important left in sub-types of `tourism=information information=*`?
What is the most important sub-type we don't explicitly support yet?

Then we can reduce `tourism-information` priority significantly.

**Comment by pastk** [comment]
Looks good!
Please make tourism-information priority same as tourism-information-board/map then.

**Comment by Jean-BaptisteC** [comment]
Can you add `Closes #5049` in your first commentary?

**Review by vng — APPROVED** [review]

File: data/categories.txt (hunk 1 above)
Comment (map-per) [reply data/categories.txt:11646]: :+1:

**Pull request merged**